guide/tutorial-4-more-about-controllers-20-minutes.md

```diff
@@ -170,7 +170,7 @@ sensors as follows:
 >   "ps4", "ps5", "ps6", "ps7"
 > };
 >
-> for (i=0; i<8; i++) {
+> for (int i=0; i<8; i++) {
```

Comment:
In the majority of C compilers, this will leads to a warning (or even errors?), because the `i` variable cannot be defined in the `for` statement:

```
int i; // define int above.
for (i=0; i<8; i++) {
```

.. and `int i;` is already defined above in the code.
Same comment for the other modifications bellow.